Comanche055/KEYRUPT_UPRUPT.agc

```diff
@@ -88,12 +88,12 @@ UPRPT1		CAF	LOW5		# TEST FOR TRIPLE CHAR REDUNDANCY
 		COM
 		TC	UPTEST
 
-UPOK		CS	ELRCODE		# CODE IS GOOD.  IF CODE = `ERROR RESET',
-		AD	KEYTEMP1	# CLEAR UPLOCKFL (SET BIT4 OF FLAGWRD7 = 0)
-		EXTEND			# IF CODE DOES NOT = `ERROR RESET', ACCEPT
```

Comment:
It used to be a habit of opening quotes with a grave accent and closing them with a single quotation mark. I suppose the same habit can be seen here as well.

Reply:
This is just an "FYI" remark, right? :)

Reply:
Yeah.